dune-project

```diff
@@ -2,3 +2,29 @@
 (name lambdapi)
 
 (using menhir 1.0)
+
+(generate_opam_files true)
+(source (github Deducteam/lambdapi))
+(license CeCILL2.1)
+(authors "Rodolphe Lepigre <lepigre@mpi-sws.org>"
+         "Frédéric Blanqui <frederic.blanqui@inria.fr>"
+         "Emilio Jesús Gallego Arias <e@x80.org>")
```

Comment:
There are contributions by other people. So it would be better to put no names at all. Contributors can be found out on github.